docs/getting-started/actions.md

```diff
@@ -77,97 +147,141 @@ config.add_page(name='Trigger Variable', content=test_page())
+        Button('Reset', onclick=reset()),
     )
 
 config.add_page(name='Reset Variable', content=test_page())
 ```
 
----
+For simple cases check out [reset](#daracoreinteractivityany_variableanyvariablereset) shortcut action.
+
+### `notify`
+
+```python
+async def notify(
+    message: str,
+    title: str,
+    status: NotificationStatusString | NotificationStatus,
+    key: str | None = None
+)
+```
 
-As mentioned, actions do not have to affect `Variable`s. They can also enable other kinds of interactivity.
+`notify` is a method to display a notification toast on the client. It takes the following arguments:
 
-## `Notify`
+- `message` - the message to display in the notification
+- `title` - the title of the notification
+- `status` - the status of the notification, can be one of members of the `NotificationStatus` enum or its string value
+- `key` - a unique key for the notification, if not provided the title will be used as such; used to uniquely identify notifications to prevent the same notification from being displayed multiple times
 
-The `dara.core.interactivity.actions.Notify` action triggers the UI to display a toast notification with a title and a message. In the example below, you can trigger a notification by clicking on the 'Notify' button.
+See the following example:
 
 ```python
-from dara.core import ConfigurationBuilder
-from dara.core.interactivity.actions import Notify, NotificationStatus
+from dara.core import action, ConfigurationBuilder
 
 from dara.components import Stack, Button
 
 config = ConfigurationBuilder()
 
+@action
+async def notify(ctx: action.Ctx):
+    await ctx.notify(message='This is the notification message', title='Example', status='SUCCESS')
 
 def test_page():
     return Stack(
         Button(
             'Notify',
-            onclick=Notify(
-                message='This is the notification message', title='Example', status=NotificationStatus.SUCCESS
-            ),
+            onclick=notify()
         )
     )
 
 
 config.add_page(name='Notify Example', content=test_page())
 ```
 
-## `NavigateTo`
+### `navigate`
+
+```python
+async def navigate(
+    url: str,
+    new_tab: bool = False
+)
+```
 
-The `dara.core.interactivity.actions.NavigateTo` action will trigger a change in route based on the url passed. The url can be a static string, or it can be a function that will be called with the element that triggered the action.
+`navigate` is a method to navigate to a new URL. It takes the following arguments:
 
-In the example below, on 'Test Page', clicking on a button 'Go to Another Page' takes you to the 'Another Page' page. This demonstrates how you can pass a static url to the `NavigateTo` action. On the 'Another Page' page, you can see how to control `NavigateTo` by passing url as a function based on component value.
+- `url` - the URL to navigate to
+- `new_tab` - a boolean indicating whether to open the URL in a new tab
+
+See the following example:
```

Comment:
Yeah I just felt like it's unnecessary and the page is already pretty detailed, with the same examples being reiterated with different APIs. If you feel like any of them in particular are crucial for making it easier to understand happy to bring them back

Reply:
I think most are fine, for the url example maybe worth having somewhere in the code example that you are navigating to a different page just in case we have someone new to coding/technical things and that don't have very clear what it means.

Reply:
Fair enough, added back the explanation for the navigate example